Package.swift

```diff
@@ -170,7 +170,7 @@ let package = Package(
     .package(
       name: "nanopb",
       url: "https://github.com/firebase/nanopb.git",
-      "2.30908.0" ..< "2.30909.0"
+      "2.30909.0" ..< "2.30910.0"
```

Comment:
Sorry, one last question. Just double checking. Is it intended to be slightly stricter for SPM. (We aren't strict in GoogleAppMeasurement manifest)?

Reply:
Yes. We want to force SPM users to get the fix, since SPM would otherwise be more indeterministic about updating. Also I would be very surprised if there are any other SDKs that would need to couple with Firebase and an old version of nanopb